MangoPay/Mandate.php

```diff
@@ -51,7 +52,7 @@ class Mandate extends Libraries\EntityBase
 
     /**
      * Status of the mandate: CREATED, SUBMITTED, ACTIVE, FAILED
-     * @var string
+     * @var \MangoPay\MandateStatus
```

Comment:
This is incorrect: it implies that the Status is an instance of `MandateStatus` which it isn't - it's still a string with one of the values from a `MandateStatus` constant. This needs to be fixed as it breaks static code analysis completely.